sys/freebsd/pf.txt

```diff
@@ -27,7 +27,6 @@ ioctl$DIOCGETRULES(fd fd_pf, cmd const[DIOCGETRULES], arg ptr[in, array[int8]])
 ioctl$DIOCGETSTATE(fd fd_pf, cmd const[DIOCGETSTATE], arg ptr[in, array[int8]])
 ioctl$DIOCSETSTATUSIF(fd fd_pf, cmd const[DIOCSETSTATUSIF], arg ptr[in, pfioc_if])
 ioctl$DIOCNATLOOK(fd fd_pf, cmd const[DIOCNATLOOK], arg ptr[in, array[int8]])
-ioctl$DIOCGETSTATES(fd fd_pf, cmd const[DIOCGETSTATES], arg ptr[in, array[int8]])
```

Comment:
These IOCTLs are deprecated but supported for backwards compat and haven't been removed yet; in any case, if they will be removed I'd expect that to be a separate pull request.
```
#ifdef COMPAT_FREEBSD14
#define DIOCGETSTATES   _IOWR('D', 25, struct pfioc_states)
...
#define DIOCGETSTATESV2 _IOWR('D', 93, struct pfioc_states_v2)
#endif
```

Reply:
Does adding `-DCOMPAT_FREEBSD14` to the compiler flags in sys/syz-extract/freebsd.go:processFile() fix this error?

Reply:
Yes, it actually does!

Reply:
Yes please! Add the define in a first commit, then regenerate syscall definitions, and then I believe you won't have to touch pf.txt at all.

Reply:
As I removed a line in pf.txt in my initial commit, I had to add the line again in my latest commit such that no modifications to pf.txt have been performed in summary.

Reply:
It would be preferable to rewrite the commit history, e.g., using `git rebase`, to avoid removing the line at all.

Reply:
I rewrote the commit history by `git rebase`. I have split the changes into two commits: first I added the `-DCOMPAT_FREEBSD14` compiler flag and second, I added the socket_bluetooth files. I hope that this is the way you wanted it to be.